aip/general/0140.md

```diff
@@ -33,6 +33,9 @@ including adjectives that always apply and add little cognitive value. For
 example, a `proxy_settings` field might be as helpful as
 `shared_proxy_settings` if there is no unshared variant.
 
+**Note:** Field names often appear in generated client surfaces. Ensure they 
```

Comment:
Let's actually use `Important`, as it gives us a yellow box which is a little more "in your face" e.g. go/aip/149 has a few
```suggestion
**Important:** Field names often appear in generated client surfaces. Ensure they 
```

Reply:
Done